tox.ini

```diff
@@ -0,0 +1,347 @@
+         --cov-report=term \
+         {posargs} \
+         {[base]pytesttarget}
+
+[testenv:py27]
+envdir = {toxworkdir}/env-{env:TRAVIS_PYTHON_VERSION:2.7}
+basepython = python2.7
+passenv = {[base]passenv}
+setenv =
+    {[base]setenv}
+changedir = {[base]changedir}
+whitelist_externals =
+    {[base]whitelist_externals}
+commands_pre =
+    bash {toxinidir}/install_tuned_for_testing.sh {envsitepackagesdir} 2.11.0
+commands =
+    bash {[base]runpytest} {envpython} \
+         --durations=5 \
+         {[base]covtargets} \
+         --cov-report=html:htmlcov-py27 \
+         --cov-report=term \
+         {posargs} \
+         {[base]pytesttarget}
+
+[testenv:py36]
+envdir = {toxworkdir}/env-{env:TRAVIS_PYTHON_VERSION:3.6}
+basepython = python3.6
+passenv = {[base]passenv}
+setenv =
+    {[base]setenv}
+changedir = {[base]changedir}
+whitelist_externals =
+    {[base]whitelist_externals}
+commands_pre =
+    bash {toxinidir}/install_tuned_for_testing.sh {envsitepackagesdir} 2.13.0
+commands =
+    bash {[base]runpytest} {envpython} \
+         --durations=5 \
+         {[base]covtargets} \
+         --cov-report=html:htmlcov-py36 \
+         --cov-report=term \
+         {posargs} \
+         {[base]pytesttarget}
+
+[testenv:py37]
+envdir = {toxworkdir}/env-{env:TRAVIS_PYTHON_VERSION:3.7}
+basepython = python3.7
+passenv = {[base]passenv}
+setenv =
+    {[base]setenv}
+changedir = {[base]changedir}
+whitelist_externals =
+    {[base]whitelist_externals}
+commands_pre =
+    bash {toxinidir}/install_tuned_for_testing.sh {envsitepackagesdir} 2.13.0
+commands =
+    bash {[base]runpytest} {envpython} \
+         --durations=5 \
+         {[base]covtargets} \
+         --cov-report=html:htmlcov-py37 \
+         --cov-report=term \
+         {posargs} \
+         {[base]pytesttarget}
+
+[testenv:py38]
+envdir = {toxworkdir}/env-{env:TRAVIS_PYTHON_VERSION:3.8}
+basepython = python3.8
+passenv = {[base]passenv}
+setenv =
+    {[base]setenv}
+changedir = {[base]changedir}
+whitelist_externals =
+    {[base]whitelist_externals}
+commands_pre =
+    bash {toxinidir}/install_tuned_for_testing.sh {envsitepackagesdir} 2.13.0
+commands =
+    bash {[base]runpytest} {envpython} \
+         --durations=5 \
+         {[base]covtargets} \
+         --cov-report=html:htmlcov-py38 \
+         --cov-report=term \
+         {posargs} \
+         {[base]pytesttarget}
+
+[testenv:black]
+envdir = {toxworkdir}/env-{env:TRAVIS_PYTHON_VERSION:3.6}
+basepython = python3.6
+passenv = RUN_BLACK_*
+deps =
+    black
+whitelist_externals =
+    {[base]whitelist_externals}
+commands =
+    bash {toxinidir}/.travis/runblack.sh {envpython} --check --diff .
+
+[testenv:pylint]
+envdir = {toxworkdir}/env-{env:TRAVIS_PYTHON_VERSION:2.7}
+basepython = python2.7
+passenv = RUN_PYLINT_*
+setenv =
+    {[base]setenv}
+deps =
+    {[testenv]deps}
+    colorama
+    pylint>=1.8.4
+    ansible
+    -rpylint_extra_requirements.txt
+    -rtuned211_requirements.txt
```

Comment:
I can't, because I want to use tuned-2.11.0 for python 2.x (roughly corresponds to el7) and tuned-2.13.0 for python 3.x (roughly corresponds to fedora and el8)

Reply:
and if I have separate files for that anyway (I need tuned for pyunit as well as pylint) I might as well keep them in separate files rather than duplicate stuff in pylint_extra_requirements.txt